src/renderer/services/plugin-manager.js

```diff
@@ -143,6 +143,19 @@ export class PluginManager {
     }
   }
 
+  async unloadLanguages (plugin, profileId) {
+    if (I18N.defaultLocale !== this.app.$store.getters['session/language']) {
+      await this.app.$store.dispatch('session/setLanguage', I18N.defaultLocale)
+
+      const profile = this.app.$store.getters['profile/byId'](profileId)
+
+      await this.app.$store.dispatch('profile/update', {
+        ...profile,
+        ...{ language: I18N.defaultLocale }
+      })
```

Comment:
Why not:

```
await this.app.$store.dispatch('profile/update', {
  ...profile,
  language: I18N.defaultLocale
})
```

instead of:

```
await this.app.$store.dispatch('profile/update', {
  ...profile,
  ...{ language: I18N.defaultLocale }
})
```

Reply:
Good question :joy: i'll change it asap